mysql/mysql.go

```diff
@@ -41,20 +41,30 @@ import (
 )
 
 const (
-	name       = "mysql"
-	version    = 7
-	pluginType = plugin.PublisherPluginType
+	name            = "mysql"
+	version         = 8
+	pluginType      = plugin.PublisherPluginType
+	usernameDefault = "root"
+	passwordDefault = "root"
+	hostnameDefault = "localhost"
+	tcpPortDefault  = "3306"
+
+	databaseDefault = "SNAP_TEST"
+	tableDefault    = "info"
+	tableColumns    = "(timestamp VARCHAR(200), source_column VARCHAR(200), key_column VARCHAR(200), value_column VARCHAR(200))"
```

Comment:
Consider reading data type for `value_column` from configuration, instead of default `VARCHAR(200)`.
Users might like to run analytics directly on the values stored in database, without parsing value first.